ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -44537,8 +44537,8 @@ periodics:
     org: openshift
     repo: openshift-tests-private
   labels:
-    ci-operator.openshift.io/cloud: powervs-1
-    ci-operator.openshift.io/cloud-cluster-profile: powervs-1
+    ci-operator.openshift.io/cloud: powervs-2
```

Comment:
I am not sure why QE test was also updated

Reply:
It is using the good pool, and the swap was meant to be consistent.